src/databricks/labs/ucx/workspace_access/scim.py

```diff
@@ -92,35 +93,13 @@ def _safe_patch_group(
     ):
         try:
             return self._ws.groups.patch(id=group_id, operations=operations, schemas=schemas)
-        except DatabricksError as e:
-            if e.error_code in [
-                "BAD_REQUEST",
-                "INVALID_PARAMETER_VALUE",
-                "UNAUTHORIZED",
-                "PERMISSION_DENIED",
-                "FEATURE_DISABLED",
-                "RESOURCE_DOES_NOT_EXIST",
-            ]:
-                logger.warning(f"Could not apply changes to group {group_id} due to {e.error_code}")
-                return None
-            else:
-                msg = f"{e.error_code} can be retried for group {group_id}, doing another attempt..."
-                raise RetryableError(message=msg) from e
+        except NotFound as e:
+            logger.warning(f"removed on backend: {group_id}")
```

Comment:
It's common practice to include the caught exception message into log along with context as this quickens the debug cycle.